tests/test_ansi.py

```diff
@@ -253,3 +253,41 @@ def test_rgb_bounds(r, g, b, valid):
             ansi.RgbFg(r, g, b)
         with pytest.raises(ValueError):
             ansi.RgbBg(r, g, b)
+
+
+def test_std_color_re():
+    """Test regular expressions for matching standard foreground and background colors"""
+    for color in ansi.Fg:
+        assert ansi.STD_FG_RE.match(str(color))
+    for color in ansi.Bg:
+        assert ansi.STD_BG_RE.match(str(color))
```

Comment:
Would it also make sense to test the reverse for each color?
Like:
```
    for color in ansi.Bg:
        assert ansi.STD_BG_RE.match(str(color))
        assert not ansi.STD_FG_RE.match(str(color))
```

Reply:
Done for all 3 functions like this